PRS6001-7000/PRS6567Makwanne.xml

```diff
@@ -51,19 +51,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
+         <change when="2019-12-18" who="DE">Added information, relations, and bibliography.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Mäkwännǝn Wäldä Mikaʾel
-               </persName>
-               <occupation> ras </occupation>
+               <persName xml:lang="gez" xml:id="n1">መኰንን፡ ወልደ፡ ሚካእል፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">Makʷannǝn Walda Mikāʾel</persName>
+               <persName><roleName type="title">bālāmbārās</roleName></persName>
+                <persName><roleName type="title">rās</roleName></persName>
+                <birth when="1852-05-08"><placeName ref="LOC3556Gola"></placeName>
+                He was a son of <persName ref="PRS9953WaldaMi"></persName> and <persName ref="PRS9235Tananna"></persName>.
+              </birth>
+                <floruit>
+                He was invited to the court of <persName ref="PRS7006Menilek"></persName> in <date>1865</date>. His first, common-law,
+                 wife gave birth to their son <persName ref="PRS10309YelmaM"></persName>. In <date>1876</date>, he married
+                 <persName ref="PRS10255Yasimma">Yašimmabet ʾAli</persName>, who gave birth to <persName ref="PRS5226haylase">Tafari</persName>.
+                 In <date>1876</date> Makʷannǝn Walda Mikāʾel became <roleName type="rank">bālāmbārās</roleName> and in
+                 <date>1881</date> he joined a campaign against the <persName ref="ETH1973Oromo"></persName>.
+                 After the conquest of <placeName ref="LOC3786Harar"></placeName>, he was awarded the rank of
+                 <roleName type="rank">daǧǧāzmāč</roleName> and became the governor of the region. In <date>1887</date>
+                 Makʷannǝn Walda Mikāʾel started the imperial expansion to the east and south of
+                 <placeName ref="LOC3786Harar"></placeName> and conquered the parts of <placeName ref="LOC5040Ogaden"></placeName>,
+                 as well as parts of the territories of <persName ref="ETH1496Gadabu"></persName> and
+                 <persName ref="ETH1728Issa"></persName>.<lb/>
```

Comment:
In the guidelines, `<lb>` is only mentioned in text encoding. Maybe a structuring in `<p>`s (to be visualized, if they are not yet...) would be more meaningful?